package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "devDependencies": {
+    "webpack": "^5.67.0",
+    "webpack-cli": "^4.9.1"
+  },
+  "name": "wagmify.me",
+  "description": "The tool makes it faster to navigate to common crypto sites — instead of going to CoinGeiko.ccom and searching for olympus, you can just type in `p ohm` and the tool will automatically redirect you.",
+  "version": "1.0.0",
+  "scripts": {
```

Comment:
Is there a command to build the main.js? / does it make sense to have our deploy pipeline build this or should we just compile main.js and commit it?